CompilerSource/assend/Background.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+import "Image.proto";
+
+message Background {
+  string name = 1;
+  int32 id = 2;
+
+  bool transparent = 3;
+  bool smoothEdges = 4;
+  bool preload = 5;
+  bool useAsTileset = 6;

```

Comment:
What we often do is extract a separate message for related concepts. I didn't expect you to start with backgrounds,  or I'd have synced with you on that, first. My intention is for these field names to mirror EDL constructs—we should keep track of that and document it as we design EGM.